### ISSUE TITLE
Gradient accumulation on the linen WMT example.

### ISSUE THREAD

**Issue by AkashGanesan** [google/flax#826]
Hey, we'd like to accumulate gradients.  Is there a barebones example (for  multi-GPU training) for this that we can follow? 

### Description of the model to be implemented
Transformer-base.  Potentially other transformer models as well.

### Dataset the model could be trained on
WMT.*

### Reference implementations in other frameworks
https://stackoverflow.com/questions/53331540/accumulating-gradients

**Comment by avital** [comment]
Looks like this was answered in #827
